RATIONALE.md

```diff
@@ -0,0 +1,51 @@
+# Rationale for api designs
+
+## Get multiple traces: /api/v2/traceMulti?traceIds=id1,id2
+Get by multiple trace IDs was added in [/api/v2](zipkin2-api.yaml) to support
+more efficient requests when trace IDs are found by means besides the query api.
+
+Here are some use cases:
+* out-of-band aggregations, such histograms with attached representative IDs
+* trace comparison views
```

Comment:
We currently don't do this, but when multiple trace IDs are possible to compare, a "refresh" could avoid making separate api requests per trace ID. That said, the efficiency of this use case could be less if `/trace/{traceId}` was cached. Hence, this is possibly the weaker use case.

Reply:
+1 this would be an awesome feature on the UI, not sure how it will look like, maybe a stacked trace timeline view would be a good start.